common/catalog/catalog.bom

```diff
@@ -22,4 +23,14 @@ brooklyn.catalog:
       iconUrl: classpath://io.brooklyn.clocker.common:icons/haproxy.png
       item:
         services:
-          - type: haproxy-load-balancer
\ No newline at end of file
+          - type: haproxy-load-balancer
+
+    - id: docker-engine-template
```

Comment:
Can you remove this template ... 
IMO its just wrapping the docker-engine entity adding no extra value

Reply:
Sure, I'll remove it